src/wallet/ClaimItem.js

```diff
@@ -0,0 +1,32 @@
+import { Fixed8 } from '../utils'
```

Comment:
Might be worth to shift these into a components folder so that the `wallet` folder exposes the top level stuff. These functions aren't meant to be used regularly by consumers.

Reply:
these `components` are currently exported from the wallet package folder
https://github.com/CityOfZion/neon-js/blob/master/src/wallet/index.js#L44

I can place them into a `components` folder, but removing them from the export would be a breaking change. If this is desired, I can updated for this.

Reply:
Place them into a folder but flatten the exports at the `wallet` level. I would like your opinion on this though. Does having too many files clutter the `wallet` folder or is it fine?

Reply:
I think it makes since to keep the components together, so put them all into a folder. Good compromise between getting too many files in the wallet folder and having all components in one file.